R/json_nest.R

```diff
@@ -0,0 +1,32 @@
+# FIXME: once tbl_lazy method is implemented, update doc of `.data` arg
```

Comment:
No need to postpone this, implementing this is on the agenda :-)

Reply:
Done in https://github.com/cynkra/dm/pull/918/commits/80af53f925d7fbd3359d353dc957a4afb17487a1